apps/src/music/views/chordPanel.module.scss

```diff
@@ -1,82 +1,113 @@
 @import "color.scss";
 
 .chordPanelContainer {
-  display: flex;
```

Comment:
just out curiosity - why not continue to use `flex` here?

Reply:
TBH, I don't fully remember any more.  There was a lot of styling iteration in this PR (with it starting as a side-by-side layout), and I guess it wasn't necessary with things simply stacked top to bottom.